### PR TITLE
Fix evergreen updateClient crash

### PR DESCRIPTION
**Pull request by sherryzy**
b/335726353

### CODE REVIEW

**Comment by sherryzy** [comment]
This PR doesn't fix evergreen tests entirely, but b/335726353 seems to be gone. The app doesn't crash after 30s any more.

File: components/update_client/net/network_impl_cobalt.cc [hunk 1]
```diff
@@ -184,25 +185,26 @@ void NetworkFetcherCobaltImpl::OnURLFetchResponseStarted(
 void NetworkFetcherCobaltImpl::OnURLFetchComplete(
     const net::URLFetcher* source) {
   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
+  LOG(INFO) << "cobalt::updater::NetworkFetcher::OnURLFetchComplete";
```
Comment (andrewsavage1) [inline components/update_client/net/network_impl_cobalt.cc:188]: nit: remove

**Review by andrewsavage1 — APPROVED** [review]
All LGTM, but please have Jelle take a look too

File: components/update_client/net/network_impl_cobalt.cc (hunk 1 above)
Comment (sherryzy) [reply components/update_client/net/network_impl_cobalt.cc:188]: Hmm, this line is not for my debugging purpose. And it's basically a copy past of this file: cobalt/updater/network_fetcher.cc. In addition, if you look at the functions in this file, each function has a LOG(INFO), so I think we should keep it here.

Comment (andrewsavage1) [reply components/update_client/net/network_impl_cobalt.cc:188]: Got it, assumed it was for debugging

**Review by jellefoks — APPROVED** [review]

**Pull request merged**